.github/workflows/connect-transport-e2e-test.yml

```diff
@@ -0,0 +1,48 @@
+name: Transport E2E Tests
+
+on:
+  schedule:
+    # Runs at midnight UTC every day at 01:00 AM CET
+    - cron: "0 0 * * *"
+  pull_request:
+    paths:
+      - "packages/transport/**"
+      - "packages/protobuf/**"
+      - "packages/protocol/**"
+      - "packages/trezor-user-env-link/**"
+      - "packages/utils/**"
+      - "docker/docker-compose.transport-test.yml"
+      - ".github/workflows/connect-transport-e2e-test.yml"
+      - "yarn.lock"
+
+jobs:
+  e2e:
```

Comment:
this should be more descriptive

Reply:
Something like: https://github.com/trezor/trezor-suite/pull/11609/commits/b52b5e0dd372879acf671166504ef658cfc20cbd  :question: